source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md

```diff
@@ -63,6 +63,14 @@ ContentChange.where("created_at < ?", 10.minutes.ago).where(processed_at: nil).m
 
 - [content_change_failed_emails][]
 
+#### Resend failed emails by email ids
+
+- [resend_failed_emails:by_id][]
+
+#### Resend failed emails by date range
+
+- [resend_failed_emails:by_date['from, to']][]
```

Comment:
This markdown has come out oddly.

Reply:
The links should be rendering correctly now. Good spot 👁️